MigrationCodelab/app/build.gradle

```diff
@@ -92,8 +92,8 @@ dependencies {
     implementation "androidx.compose.foundation:foundation-layout:$rootProject.composeVersion"
     implementation "androidx.compose.material:material:$rootProject.composeVersion"
     implementation "androidx.compose.runtime:runtime-livedata:$rootProject.composeVersion"
-    implementation "androidx.ui:ui-tooling:$rootProject.composeVersion"
-    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
+    implementation "androidx.compose.ui:ui-tooling:$rootProject.composeVersion"
+    implementation "com.google.android.material:compose-theme-adapter:1.0.0-alpha09"
```

Comment:
Will alpha09 of the theme adapter not conflict with the alpha10 release? Just double checking :)

Reply:
Nope, there are no breaking changes from alpha09 so it's safe to use. This will use the same version of Compose next week when it's released